src/components/CustomButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+import "../styles/Button.css";
+
+function CustomButton() {
```

Comment:
Rename to SubmitButton... CustomButton is nondescript - it is important to write self-documenting code